rero_ils/modules/loans/jsonschemas/loans/loan-ils-v0.0.1.json

```diff
@@ -90,6 +90,10 @@
       "type": "string",
       "title": "Transaction location PID"
     },
+    "checkout_location_pid": {
+      "type": "string",
```

Comment:
`minLength` is missing.

Reply:
Seems a little bit overkill, isn't it ? 
Sure I could, but then, to be consistant, I would update all "_pid" field from all schema (in this schema I count 9 fields to update in this way)